ducktape/unit_tests/tests/check_loader.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2015 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.tests.loader import TestLoader
+
+import os
+
+
+class CheckTestLoader(object):
+
+    # Assume base_dir is the unit_test directory
+    BASE_DIR = os.path.abspath(os.path.curdir)
```

Comment:
I noticed you can't run the tests from another directory, and I'm assuming this is why. It'd be better to get this `BASE_DIR` variable dynamically, either from pytest or assuming an offset from this file.